### PR TITLE
Plane: separate out landing adjustment from baro offset

### PR DESCRIPTION
**Pull request by tridge**

### CODE REVIEW

File: ArduPlane/altitude.cpp [hunk 1]
```diff
@@ -464,6 +464,24 @@ int32_t Plane::adjusted_relative_altitude_cm(void)
...
+/*
+  return the mission altitude offset. This raises or lowers all
+  mission items. It is primarily set using the ALT_OFFSET parameter,
+  but can also be adjusted by the rangefinder landing code for a
+  NAV_LAND command if we have aborted a steep landing
+ */
+float Plane::mission_alt_offset(void)
+{
+    float ret = g.alt_offset;
+    if (control_mode == AUTO && mission.get_current_nav_cmd().id == MAV_CMD_NAV_LAND) {
```
Comment (magicrub) [inline ArduPlane/altitude.cpp]: instead of checking mission id, it should use auto_state.land_in_progress

Comment (tridge) [inline ArduPlane/altitude.cpp]: fixed

**Comment by magicrub** [comment]
Lgtm. I'll do a SITL test in the morning to confirm the offset polarity.

**Comment by tridge** [comment]
thanks Tom!

**Comment by magicrub** [comment]
Merged

**Pull request closed without merging**